org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/extensions/IRenameResponse.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+* Copyright (c) 2023 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package org.eclipse.lemminx.services.extensions;
```

Comment:
Remove IRenameParticipant, IRenameRequest, IRenameParticipant in the package `org.eclipse.lemminx.extensions.rename`

Reply:
I don't understand what should I do here, could you explain, please? 
Or this change is outdated and I have already cleaned up the imports...

Reply:
Oops sorry yes it is that like other participant hover etc

Reply:
Done